tests/unit/tracking.tcl

```diff
@@ -194,12 +197,17 @@ start_server {tags {"tracking network"}} {
         $rd_sg SET key1 1
         r GET key1
         $rd_redirection QUIT
+        # Next SET may be send to server with QUIT together
+        $rd_redirection read
```

Comment:
considering the type of bug we fix here, and the fact responses multiple clients mixed into one connection (redirection), i think we should make an effort to validate each response to the expected value, maybe we'll be able to catch more bugs.

if you agree, let's add `assert_equals` to all the `read` calls, and some of the other commands (maybe `r GET` and such).

Reply:
agree. I think it's a good idea to improve this test. Unlike other commands, client tracking will involve multiple clients and server push, the test will be more complex.

Reply:
@huangzhw will you have time to improve that in this PR? or shall we merge it in the current form?

Reply:
@oranagra I will take it.